tensorrt_llm/_torch/modules/linear.py

```diff
@@ -766,9 +766,14 @@ def apply(self, module: Linear, input: torch.Tensor,
             act_fp4, act_sf = torch.ops.trtllm.fp4_quantize(
                 input, module.input_scale, module.scaling_vector_size, False)
 
-        output = torch.ops.trtllm.nvfp4_gemm(act_fp4, module.weight, act_sf,
-                                             module.weight_scale, module.alpha,
-                                             module.dtype)
+        if module.use_cute_dsl_nvfp4_blockscaling_mm:
```

Comment:
Not necessarily in this PR.

But @hyukn  @limin2021  do you think we should/can autotune between cutedsl and cutlass fp4 gemm?